### DISCUSSION TITLE
What do we think about GitHub discussions?

### DISCUSSION THREAD

**Discussion by jheek** [google/flax#327] [General] [answered]
What do we think about GitHub discussions?

**Comment by avital** [comment]
Let's find out!

**Reply by danielsuo** [reply]
Hadn’t noticed this even get introduced!

Pro: separate space for longer-form discussion
Con: another space for longer-form discussion

Not sure I had much to add...

**Reply by avital** [reply]
Yeah we just got added to the beta. I'd like to modify the README to encourage people to have discussions here.

**Reply by danielsuo** [reply]
How should people think about issues vs discussion? Is it bugs / feature requests vs questions? Or external vs devs? Something else? Not sure how best to use yet, but this is neat!

**Comment by avital** [comment]
And the threading feature seems nice

**Reply by dynamicwebpaige** [reply]
Is is possible to reference PRs or issues? #326

**Reply by dynamicwebpaige** [reply]
- [x] Looks
- [x] Like
- [x] Yes!

**Comment by dynamicwebpaige** [comment, marked as answer]
#### Does this thingum support markdown?
Looks like a yes!

#### What about GIFs?
![tenor (5)](https://user-images.githubusercontent.com/3712347/84977236-b559a200-b0de-11ea-8512-0feae91b741c.gif)

**Reply by avital** [reply]
Can I continue replying after I marked an answer as correct?

**Reply by avital** [reply]
Looks like yes. But it there some way to block additional comments if a correct answer is posted, so that people don't miss it?

**Reply by avital** [reply]
There doesn't seem to be a way to do that. There's a risk that the selected answers get drowned away with subsequent comments -- let's see if that becomes an issue.

**Reply by avital** [reply]
I guess even with many comments that are collapsed, as long as the answer itself is correct it will always be visible so that's probably fine...

**Reply by jheek** [reply]
You can lock the conversation right?

**Reply by dynamicwebpaige** [reply]
> Is there some way to block additional comments if a correct answer is posted, so that people don't miss it?

This could also be solved by having the endorsed solution pinned to the top of the question. Sounds like a feature request! 😄

**Reply by becca** [reply]
👋 we are working on making it easier to `jump to` the answer when it's selected 😄 so this will be coming soon!

**Reply by 8bitmp3** [reply]
@becca If we click on **Answered ⬇️** in the top left corner, the web UI takes us straight to the answer. Very awesome.

![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia1.tenor.com%2Fimages%2F0f99c3b8818fc1abace8b8b3cc60898a%2Ftenor.gif)
